package.json

```diff
@@ -60,6 +60,7 @@
     "@babel/plugin-syntax-import-meta": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
+    "babel-plugin-styled-components": "^1.10.6",
```

Comment:
Was there something happening that this is fixing?

Reply:
I added it so we could have access to these features. 

https://www.styled-components.com/docs/tooling#usage